.github/workflows/stale-issue.yml

```diff
@@ -0,0 +1,55 @@
+# GitHub action based on https://github.com/actions/stale
+
+name: 'Close stale issues and PRs'
```

Comment:
There is now a top-level `permissions` block which Github suggests adding to all workflows. Check out our other workflows.

Reply:
Actually there is a suggested permission list: https://github.com/actions/stale#recommended-permissions

Reply:
Added